Services/Interfaces/web_interface/templates/backtesting.html

```diff
@@ -53,6 +53,9 @@ <h2>Backtesting
                 {% endfor %}
               </tbody>
             </table>
+            <div class="d-flex justify-content-end">
+                <a href="{{ url_for('data_collector') }}" class="btn btn-outline-info btn-lg waves-effect"> <i class="fa fa-cloud-download-alt"></i> Get historical data</a>
```

Comment:
```suggestion
                <a href="{{ url_for('data_collector') }}" class="btn btn-info waves-effect"> <i class="fa fa-cloud-download-alt"></i> Get historical data</a>
```
![image](https://user-images.githubusercontent.com/9078616/139599398-7ea1eda3-52fa-435e-85c3-dea7b7dfc88d.png)

Reply:
done

Reply:
great, thanks !